engine/api/src/org/apache/cloudstack/engine/orchestration/service/NetworkOrchestrationService.java

```diff
@@ -62,9 +63,13 @@
     static final String NetworkLockTimeoutCK = "network.lock.timeout";
     static final String GuestDomainSuffixCK = "guest.domain.suffix";
     static final String NetworkThrottlingRateCK = "network.throttling.rate";
+    static final String MinVRVersionCK = "minreq.sysvmtemplate.version";
+
+    static final ConfigKey<String> MinVRVersion = new ConfigKey<String>(String.class, MinVRVersionCK, "Advanced", "4.4.4",
```

Comment:
No,  4.4.2 is the minimum as some fixes went into the 4.4 branch.  I don't mind that instead of 4.4.4 that I did because I am an irritatingly formal guy.